update/errors.go

```diff
@@ -0,0 +1,15 @@
+package update
```

Comment:
Comments missing (also in the other files, but I figure they will come after the architecture for this will take shape)

Reply:
yes. next PR will resolve a lot.